package/MDAnalysis/analysis/base.py

```diff
@@ -555,23 +576,23 @@ def _configure_backend(
             unsupported_backend: bool = False
             ) -> BackendBase:
         """Matches a passed backend string value with class attributes
-        :meth:`_is_parallelizable()` and :meth:`get_supported_backends()`
+        :meth:`parallelizable` and :meth:`get_supported_backends()`
```

Comment:
I _think_ we need to mark-up managed attributes as attr instead of meth
```suggestion
        :attr:`parallelizable` and :meth:`get_supported_backends()`
```